app/src/main/java/com/framgia/englishconversation/screen/CustomSimpleExoPlayerHelper.java

```diff
@@ -0,0 +1,96 @@
+package com.framgia.englishconversation.screen;
+
+import android.net.Uri;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.google.android.exoplayer2.ParserException;
+import com.google.android.exoplayer2.ui.SimpleExoPlayerView;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import im.ene.toro.ToroPlayer;
+import im.ene.toro.exoplayer.ExoPlayerHelper;
+import im.ene.toro.helper.ToroPlayerHelper;
+import im.ene.toro.media.PlaybackInfo;
+import im.ene.toro.widget.Container;
+
+/**
+ * Created by fs-sournary.
+ * Data: 1/18/18.
+ * Description:
+ */
+
+public final class CustomSimpleExoPlayerHelper extends ToroPlayerHelper {
+
+    private final ExoPlayerHelper.EventListener internalListener =
+            new ExoPlayerHelper.EventListener() {
+                @Override
+                public void onPlayerStateChanged(boolean playWhenReady, int playbackState) {
+                    CustomSimpleExoPlayerHelper.super.onPlayerStateUpdated(
+                            playWhenReady, playbackState);
+                    super.onPlayerStateChanged(playWhenReady, playbackState);
+                }
+            };
```

Comment:
ERROR: 'object def rcurly' have incorrect indentation level 12, expected level should be one of the following: 4, 8.